### PR TITLE
Contributing in MD

### PR DESCRIPTION
**Pull request by Jetrom17**
Captura de tela de 2024-10-22 11-35-33

### CODE REVIEW

**Review by trindadedev13 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Once you've forked the repository, clone it to your local machine:
+
+```bash
+git clone https://github.com/SparklesIDE/Sparkles-app.git
+cd Sparkles-app
+```
+
+#### 3. Create a New Branch (Optional)
+Create a new branch for your feature or bug fix:
+git checkout -b feature/your-feature-name
```
Comment (trindadedev13) [inline CONTRIBUTING.md:20]: wouldn't it be git branch -M your-feature-name?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+> [!Warning]
+> Have you been blocked by Github on your machine? Github's new policy requires an authentication token. Once you get it, you can continue as normal, but with a token. This usually affects when you try to upload to your repository and not to the contributor.
+> https://github.com/settings/tokens/new
+
+#### 8. Open a Pull Request
+Go to the original repository where you want to propose your changes. Click on the "Pull Requests" tab and then click the "New Pull Request" button. Select your branch and submit the pull request.
+
+> [!NOTE]
+>  ### Additional Guidelines
+> - **License**: Please adhere to our [License](license).
```
Comment (trindadedev13) [inline CONTRIBUTING.md:56]: what

File: CONTRIBUTING.md (hunk 1 above)
Comment (Jetrom17) [reply CONTRIBUTING.md:20]: It is to create a new one and change branch at the same time.

Great for isolating the original branch. Your suggestion is to rename existing branch. I do not recommend it.

https://git-scm.com/docs/git-checkout/pt_BR

File: CONTRIBUTING.md (hunk 2 above)
Comment (Jetrom17) [reply CONTRIBUTING.md:56]: Just to be aware it is in accordance with the license.

File: CONTRIBUTING.md (hunk 1 above)
Comment (trindadedev13) [reply CONTRIBUTING.md:20]: ok

**Review by trindadedev13 — APPROVED** [review]
lgtm

**Pull request merged**